xc/xc7/toolchain_wrappers/synth

```diff
@@ -4,12 +4,17 @@ set -e
 MYPATH=`realpath $0`
 MYPATH=`dirname ${MYPATH}`
 
-export TECHMAP_PATH=`realpath ${MYPATH}/../share/techmaps/xc7_vpr/techmap`
+export TECHMAP_PATH=`realpath ${MYPATH}/../share/symbiflow/techmaps/xc7_vpr/techmap`
```

Comment:
I was wondering if it should be;
 * `share/symbiflow/xc7/techmaps/`
 * `share/symbiflow/xc7/scripts/`
 * `share/symbiflow/xc7/<blah>/`

Rather than;
 * `share/symbiflow/scripts/xc7`
 * `share/symbiflow/tecmaps/xc7`
 * etc....

Reply:
Probably easier to understand as you suggest, I'll change that